README.md

```diff
@@ -305,12 +317,15 @@ header. (Use the value `application/vnd.google.protobuf;
 proto=io.prometheus.client.MetricFamily; encoding=delimited` for protocol
 buffers, otherwise the text format is tried as a fall-back.)
 
-The response code upon success is either 200 or 400. A 200 response implies a
-successful push, either replacing an existing group of metrics or creating a
-new one. A 400 response can happen if the request is malformed or if the pushed
-metrics are inconsistent with metrics pushed to other groups or collide with
-metrics of the Pushgateway itself. An explanation is returned in the body of
-the response and logged on error level.
+The response code upon success is either 200, 202, or 400. A 200 response
```

Comment:
I wonder if 202 will catch existing custom clients off guard (naively expecting 200).

Returning 200 isn't incorrect, so it's a question of value and whether any client would actually need to discern this (I doubt it).

Reply:
202 is what we had before the push checks got introduced. So all clients, custom or not, that worked with PGW v0.9 and earlier should also work with this. 

My assumption here is therefore that all the existing clients are fine with it, and we get the differentiation of the two different modes for free.